src/Model.elm

```diff
@@ -253,6 +257,20 @@ update msg model =
                 Err error ->
                     { model | userTimeline = [], errors = (errorText error) :: model.errors } ! []
 
+        OnLoadUserAccount accountId ->
+            {-
+               @TODO
+               When requesting a user profile, we should load a new "page"
```

Comment:
Well in fact it's really easy to do. https://github.com/elm-lang/navigation and https://github.com/evancz/url-parser are insanely easy to use. I just love to put @TODO everywhere when I on the subject :-)